buildtools/webpack/plugins.js

```diff
@@ -79,7 +87,7 @@ function plugins(options = {}) {
     return [ emptyModule(), uglify(), banner() ];
   }
   // Use DEBUG/development environment w/ console warnings
-  return [ emptyModule(), devMode() ];
+  return [ emptyModule(), devMode(), webpackBundleAnalyzer() ];
```

Comment:
Any reason why this is only generated for dev builds?

Reply:
good question.
1. `release` build generates 4 bundle files which means need further refactor to the build script.
2. I'm not sure if we want to analyze release build at this moment given dev build already gives a lot insights.
2. adds a little time to release build

Reply:
I've never seen the measurements differ by more than a few percent.

ScaleFT frontend prod build: ![screenshot_2018-10-02 webpack bundle analyzer 1](https://user-images.githubusercontent.com/41267141/46375638-cfbcc500-c648-11e8-872f-1bbb166fb43c.png)

ScaleFT frontend dev build: ![screenshot_2018-10-02 webpack bundle analyzer](https://user-images.githubusercontent.com/41267141/46375639-cfbcc500-c648-11e8-93bf-b2313bd96104.png)

Reply:
iirc, the diff is about the https://www.npmjs.com/package/webpack-bundle-analyzer#parsed
when hook to any files